Usgs/earthquake eventpages#226

jdbrown-USGS opened the pull request:
fixes usgs/earthquake-eventpages#226

- Clicking on DYFI static map goes to interactive map with DYFI layers open.

jmfee-usgs commented:
Generally looks good.  A few changes:

DYFIPage.js
- streamline `_createTabListImage`
  - either create an `a` element or an `img` element, not both
  - inline _createSvgImage method, it's not used in multiple places
  - remove `dataObject` parameter, just reference `MAP_GRAPH_IMAGES`
- update tablist tabs to include `onDestroy` that removes event listeners; this may require a little extra refactoring.

_DYFIPage.scss
- move `.svgimagemap` css rule to svgimagemap project.

jdbrown-USGS commented:
- Streamlined _createTabListImage.
- Updated Destroy and created onDestroy to remove event listeners.
- Moved .svgimagemap css rule to svgimagemap project which as been merged.

- Same pull request
